src/codeu/chat/common/User.java

```diff
@@ -50,13 +50,14 @@ public User read(InputStream in) throws IOException {
 
   public final Uuid id;
   public final String name;
+  public final String password;
```

Comment:
Something to think about: how are passwords stored? Are you storing them as raw text in the database, or are you doing some kind of hashing?